src/index.js

```diff
@@ -35,17 +31,19 @@ export default class SplunkEvents {
     });
   }
 
-  logEvent(event) {
+  logEvent(host, level, type, workflowType, workflowInstance, event) {
```

Comment:
I agree with @alejandro-osorio when you are logging you must explicit say what are the level, type, workflowtype and workflowinstance of the log that you want to generate